### PR TITLE
Deployment monitoring updates the "from pods" list during the rollout

### PR DESCRIPTION
**Pull request by BrianBland**

### CODE REVIEW

**Comment by volkangurel** [comment]
:+1:

File: api/deployer.go [hunk 1]
```diff
@@ -423,8 +423,12 @@ func (d *deployerSpec) monitorRollout() error {
 		if err != nil {
 			return err
 		}
+		fromRunningCount := 0
+		if d.fromReplicaSet != nil {
+			_, fromRunningCount, _ = d.refreshReplicaSetPodCount(d.fromReplicaSet)
+		}
 		log.Debugf("Replica Count: Ready: %v, Running: %v", readyCount, runningCount)
-		if int32(readyCount) == *d.toReplicaSet.Spec.Replicas {
+		if readyCount == d.deployment.DesiredReplicas && fromRunningCount == 0 {
```
Comment (philk) [inline api/deployer.go]: How's `readyCount` getting populated/updated? (Asking due to the way kubectl lists the deployments)

Comment (philk) [inline api/deployer.go]: Nevermind, I could just read the code and see this is in a for loop that handles this 😁

**Comment by philk** [comment]
:+1:

**Pull request merged**